suse_caasp

```diff
@@ -349,3 +349,22 @@ if check_rpm cri-o && [ -e /usr/bin/crictl ] && [ -S /var/run/crio/crio.sock ];
     log_cmd $OF 'journalctl -u crio'
 fi
 plugin_message Done
+
+#############################################################
+section_header "Checking GPU information ..."
+OF=gpu.txt
+plugin_message "All data stored in $OF"
+
+NVIDIA_SMI="/usr/bin/nvidia-smi"
+NVIDIA_CONTAINER_CLI="/usr/bin/nvidia-container-cli"
+if [ -x "$NVIDIA_SMI" ]
+then
+    log_cmd "$OF" "$NVIDIA_SMI -q"
+    if [ -x "$NVIDIA_CONTAINER_CLI" ] 
+    then
+        log_cmd "$OF" "$NVIDIA_CONTAINER_CLI" -k -d /dev/stderr --user=root:video info"
```

Comment:
There's an extra double-quote here which will definitely result in weird errors when run. :)
```suggestion
        log_cmd "$OF" "$NVIDIA_CONTAINER_CLI -k -d /dev/stderr --user=root:video info"
```